general/clients/clients.md

```diff
@@ -191,6 +203,17 @@ This will help keep your media libraries up to date without waiting for a period
 
 **Note: Kodi's default skin does not display all unicode characters. To display unicode characters the skins font must be changed**
 
+## LG WebOS
+
+### Jellyfin for WebOS
+
+The official Jellyfin WebOS app.
+
+**Status:** ⭐ Active
+
+**Links:**
+
+* [GitHub](https://github.com/jellyfin/jellyfin-webos)
```

Comment:
newline here

Reply:
Where should I be adding a newline? It looks good to me

Reply:
nevermind I read the diff wrong